examples/seismic/abc_methods/02_damping.ipynb

```diff
@@ -0,0 +1,1257 @@
+      "text/plain": [
+       "<Figure size 1152x576 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "graph2d(u.data[0,:,:])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the solution obtained here has a reduction in noise when compared to the results displayed on the notebook <a href=\"01_introduction.ipynb\">Introduction to Acoustic Problem</a>. To plot the result of the Receivers we use the *graph2drec* routine."
```

Comment:
For the plot being references, would it be worth using the same scale as the previous plot to emphasize the reduction e.g. values now between ~ +/- 2e-01 instead of 6e-01.

Reply:
But if you look at the displacement graph it is related to the displacement in the last simulation time, while the receivers are evaluated over the entire time grid. The difference in values is natural in this case and arises from the fact that reflections over time are attenuated until the end time. Do you agree?!

Reply:
Sorry, by previous plot I meant the equivalent plot in the previous notebook (not the previous plot in the current notebook).

Reply:
The maximum and minimum values are repeated but if we looked for the reflections on the boundaries, these reflections disappear when we used the more robust ABC methods. The maximum and minimum values arise near top.